Sources/Testing/Testing.docc/LimitingExecutionTime.md

```diff
@@ -34,13 +33,13 @@ func serve100CustomersInOneHour() async {
 }
 ```
 
-If the above test function takes longer than 60 &times; 60 seconds (i.e. one
-hour) to execute, the task in which it is running will be
+If the above test function takes longer than an
+hour (60 x 60 seconds) to execute, the task in which it's running is
```

Comment:
The letter "x" here is incorrect.

Reply:
We typically change multiplication signs to plain x's.

Reply:
The typesetting lobe of my brain doesn't love that. Glad I never made a career out of it.

Okay, if it's consistent with what's done elsewhere, I can't really complain.